public/api-builder/angular.io-package/templates/directive.template.html

```diff
@@ -0,0 +1,37 @@
+{% include "lib/githubLinks.html" -%}
+{% include "lib/paramList.html" -%}
+{% extends 'layout/base.template.html' -%}
+
+{% block body %}
+include ../../_util-fns
+p.location-badge.
+  exported from {@link {$ doc.moduleDoc.id $} {$doc.moduleDoc.id $} }
+  defined in {$ githubViewLink(doc) $}
+
+:markdown
+{$ doc.description | indentForMarkdown(2) | trimBlankLines $}
+
+.l-main-section
+:markdown
+  ## Directive Information
+
+  ### Selector
+  `{$ doc.directiveOptions.selector $}`
+
+  ### Property Bindings
```

Comment:
The "binding" word doesn't get good press recently as it is really overloaded (currently we can have property bindings, host bindings, bindings in DI and each of them means a different things despite using the same name).

I would say let's change it to just `Properties`